src/views/Fish.vue

```diff
@@ -36,65 +24,58 @@
               <th title="The maximum weight possible for this fish">Max Weight</th>
             </tr>
           </thead>
-          <tbody v-for="fish in this.fishes" :key="fish.name">
-            <tr class="color1">
+          <tbody v-for="(fish, index) in this.fishes" :key="fish.name">
+            <tr :class="'color' + ((index % 2) + 1)">
```

Comment:
template string? we also didn't bring the color classes over. should just add a `striped` class to the table to make it striped.